docs/reference-guides/rancher-manager-architecture/rancher-server-and-components.md

```diff
@@ -20,6 +20,14 @@ A high-availability Kubernetes installation is recommended for production.
 
 A Docker installation of Rancher is recommended only for development and testing purposes. The ability to migrate Rancher to a high-availability cluster depends on the Rancher version.
 
+The resources that Rancher deploys when we go as a single node Rancher deployment 
+
+![Rancher Components ](https://user-images.githubusercontent.com/37767537/233318973-880855fa-5271-468e-95eb-e1290d5c91c3.png)
```

Comment:
To prevent the images from getting out of date, I think it would be a better idea to remove the screenshots and describe how to view the components after you've deployed them.